### ISSUE TITLE
Visual Question Answering

### ISSUE THREAD

**Issue by hauntedcupoftea** [projectayre/ayre#5]

**Comment by hauntedcupoftea** [comment]
Current candidate is a [Graphcore/vqa](https://huggingface.co/datasets/Graphcore/vqa/tree/main) dataset on huggingface. This will require massive datasets that will not be uploaded to the repo.

**Comment by hauntedcupoftea** [comment]
Okay, Graphcore/vqa looks to either be very new or very niche. Document about this is almost impossible to find. Size is, i think a around 33 gigabyes in total. Should be fine for research.

**Comment by hauntedcupoftea** [comment]
decide this by Saturday.

**Comment by hauntedcupoftea** [comment]
Finalizing the Graphcore/vqa dataset. I will find a way to use custom attention masks for pretrained transformers or build from scratch if I have to.
